libr/anal/jmptbl.c

```diff
@@ -192,6 +192,7 @@ R_API bool try_walkthrough_jmptbl(RAnal *anal, RAnalFunction *fcn, RAnalBlock *b
 		return false;
 	}
 	bool is_arm = anal->cur->arch && !strncmp (anal->cur->arch, "arm", 3);
+        bool is_x86 = anal->cur->arch && !strcmp (anal->cur->arch, "x86");
```

Comment:
wrong indentation, and use `strncmp()` like `is_arm` above and check `is_arm` before calling `strncmp()`.

```suggestion
    bool is_x86 = is_arm? false: !strncmp (anal->cur->arch, "x86", 3);
```